src/ui/dialogs/exstatus.ts

```diff
@@ -0,0 +1,66 @@
+import Dialog from '../Dialog'
+import Client from '../../Client'
+
+let exstatusTemplate = require('../../../template/exstatus.hbs');
+
+let dialog: Dialog;
+
+export default function() {
+
+   let content = exstatusTemplate();
+
+   dialog = new Dialog(content);
+   let dom = dialog.open();
+
+   $(dom).ready(function(){
+      getStatus();
+   });
+
+   $('.jsxc-js-save').on('click',sendStatus);
+   $('.jsxc-js-clear').on('click',removeStatus);
+}
+
+function sendStatus()
+{
+
+   let account = Client.getAccountManager().getAccounts()[0];
+   let statustext = dialog.getDom().find('textarea[name="extended_status_text"]').val().toString();
+
+   account.getContact().setStatus(statustext);
+   let targetPresence = Client.getPresenceController().getTargetPresence();
+   let con = account.getConnection();
+   con.sendPresence(targetPresence,statustext);
+
+   account.getConnection().getPEPService().publisStatus(statustext).then(function(result) {
+         dialog.close();
+   }).catch(function (msg){
+         dialog.close();
+   });
+}
+
+function getStatus()
+{
+     let account = Client.getAccountManager().getAccounts()[0];
```

Comment:
We should check all accounts and than take the first one.

Reply:
added account choice